### PR TITLE
Selection sort rust

### LINKED ISSUE #2154: Selection Sort
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [ ] Dart - @pooja-git11
- [x] Go
- [x] Java
- [x] JS
- [x] Kotlin
- [x] Php - @BhardwajShweta25
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @raksha009
- [ ] TypeScript - @lakshyajit165

### PR DESCRIPTION
**Pull request by topcoder69**
Fixes #2154

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Coded Selection sort in Rust
- 

#### Languages Used:
- Rust
- 

#### Files Added:
- Selection_sort.rs
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by topcoder69** [comment]
Please review this. It has been a week.

**Comment by asha15** [comment]
please remove the irrelevant files and let me know

**Comment by topcoder69** [comment]
@asha15  Done.

File: Arithmetic_Progression/Arithmetic_Progression.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (asha15) [inline Arithmetic_Progression/Arithmetic_Progression.php:1]: Remove this file

Comment (topcoder69) [reply Arithmetic_Progression/Arithmetic_Progression.php]: Done

**Comment by asha15** [comment]
please squash your commits we are good to merge

**Comment by topcoder69** [comment]
> please squash your commits we are good to merge

I am sorry I didn't get it!?

**Comment by asha15** [comment]
your branch has four commits nah? to make into one commit we are using squash technique. 

you can use  `git rebase -i HEAD~<number of the commits>` command

**Comment by MastersAbh** [comment]
Hi @topcoder69 are you working on this ? 
Please read about squashing commits and do the needful.

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this PR.

**Pull request closed without merging**